packages/enzyme-test-suite/test/ShallowWrapper-spec.jsx

```diff
@@ -1722,7 +1722,7 @@ describe('shallow', () => {
     });
 
     it('finds LazyComponent when render component wrapping lazy component', () => {
-      const LazyComponent = lazy(fakeDynamicImport(DynamicComponent));
+      const LazyComponent = lazy(() => fakeDynamicImport(DynamicComponent));
```

Comment:
why are these test changes needed?

Reply:
I changes to like user cases example in [react docs](https://reactjs.org/docs/code-splitting.html#reactlazy).

```js
const OtherComponent = React.lazy(() => import('./OtherComponent'));
```

> React.lazy takes a function that must call a dynamic import(). This must return a Promise which resolves to a module with a default export containing a React component.

Reply:
I'm concerned about changing existing tests; instead, let's leave these unchanged, and also *add* test cases with the thunk?

Reply:
I viewed `lazy` test of react [here](https://github.com/facebook/react/blob/b979db4e7215957f03c4221622f0b115a868439a/packages/react-reconciler/src/__tests__/ReactLazy-test.internal.js#L37). The old test is not correct with user case of react `lazy` but I don't know it still passed. However, I think we should update it!